catroidTest/src/org/catrobat/catroid/uitest/ui/activity/ProjectActivityTest.java

```diff
@@ -361,6 +361,9 @@ public void testBackgroundSprite() {
 		solo.waitForText(solo.getString(R.string.new_project_dialog_title));
 
 		enterTextAndCloseDialog(sometext);
+		assertTrue("dialog not loaded in 5 seconds",
```

Comment:
`asserts` should be in an own section :arrow_right:  add blank line before and after `assert`